platform/core/README.md

```diff
@@ -9,10 +9,14 @@ Logic shared across polyPod platforms - ios, android, wasm.
 
 ## Building
 
-On most systems, you can simply run `make`, and that will handle everything.
-
 It is required to run `make` in a bash terminal which contains the necessary dependencies - curl, make, unzip. MacOS and Linux systems terminal should already have everything needed. On Windows, a terminal like [GitBash](https://gitforwindows.org) is needed.
 
+If you wish to only build the core binaries for your current operating system,
+run `make core`. If you only wish to build the Android binaries and bindings,
+run `make android_bindings`. Similarly, to only build the iOS binaries and
+bindings, run `make ios_bindings`.
```

Comment:
Maybe clarify that you can only run both in MacOS, since ios_bindings can't work in Linux or Windows, if I'm not wrong.

Reply:
Sure I can, at the same time `make` will return early with an error telling that only macOS can be used.

Reply:
Sure I can, at the same time `make` will return early with an error telling that only macOS can be used.